test/test_evaluation/test_train_evaluator.py

```diff
@@ -63,6 +67,7 @@ def setUp(self):
         dummy_pred_files = [os.path.join(self.ev_path, str(n)) for n in range(100, 200)]
         dummy_cv_model_files = [os.path.join(self.ev_path, str(n)) for n in range(200, 300)]
         backend_mock = unittest.mock.Mock()
+        backend_mock.temporary_directory = tempfile.gettempdir()
```

Comment:
Are all of these necessary given that the backend mock has a temporary directory itself?

Reply:
In this particular case, the backend mock doesn't have a temporary directory and that is the reason why I added this. I have unified the backend mocks that could be unified under evaluation utils, but for sure it will be cleaner when we move to pytest fixtures.

Reply:
Thanks. Yes, I'm also looking forward to using pytest fixture, but I'm afraid that has to wait a bit more...